docs/PacketFence_User_Guide.asciidoc

```diff
@@ -0,0 +1,422 @@
+
+While creating the RADIUS server certifcate make sure to have the *Extended key usage: servAuth*.
+
+Under the section `tls-config tls-common`, search for `private_key_file', `certificate_file' and `ca_file'. Those should contain respectively the path of:
+
+ * the private key for your PacketFence server,
+ * the server certificate issued by your CA for your PacketFence server,
+ * the public key of your CA.
+
+If you have an `OCSP' capable PKI you can configure it in the section `OSCP` in the eap.conf file.
+
+Lastely you will need to restart RADIUS to ensure the use of the new configuration and certificates. Please do the following:
+
+ * bin/pfcmd configreload hard
+ * bin/pfcmd service radiusd restart
+
+Make sure everything happens without errors.
+
+Now that your RADIUS is ready to handle EAP-TLS, configure your SSID connection profile on the `corporate' device using this method. Generate a client certificate for your device and install it on.  
+
+Please configure an EAPTLS source which can be found while adding a new sources under `Internal->`, simply give it a name, a description and a catch-all rule. This will allow you to validate the authentication via EAP-TLS.
+
+You can now create a new Portal Profile for EAP-TLS. Under the tab configuration, section `Policies and Access Control->Connection Profiles`, `Add profile` and select as a filter the Sub Connection Type as EAP-TLS, add your source EAP-TLS. Check the box "Automatically register devices". 
+
+You know have a full flow working for your corporate devices.
+
+Provisionning certificate
+~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Let's now imagine that we want to automate the certificate process, for enrolling new devices where a certificate cannot be pushed via GPO. This is where the provisioning will act.
+
+PKI
+^^^
+
+You will need to start by adding your PKI provider, at the moment we are able to integreate with SCEP(MSPKI), PacketFence-PKI and local PKI.
+
+For the configuration of SCEP please refer to: https://packetfence.org/doc/PacketFence_MSPKI_Quick_Install_Guide.html
+
+For the configuration of PacketFence-PKI please refer to: https://packetfence.org/doc/PacketFence_PKI_Quick_Install_Guide.html
+
+After finishing configure your PKI in PacketFence we will now do the provisioners.
+
+Provisioning
+^^^^^^^^^^^^
+
+Provisioners allow devices to automatically configure themselves to connect to the proper SSID (if applicable), use the proper authentication method (e.g. EAP-TLS) and trust the CA certificate and any certificate signed by it.
+
+Provisioners are configured in the PacketFence administration GUI under `Advanced Access Configuration->Provisioners`.
+
+Add a new provisioner for each of the device's calsses to be supported amongst Android, Apple Devices and Windows.
```

Comment:
calsses -> classes

Reply:
No need to correct typos on this PR. Let's just take what we want from it and scrap the rest.